index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <title>David Leiva</title>
+  </head>
+  <body>
+
+    <!-- Main navigation bar or toolbar -->
+    <nav>
+      <div class="logo"><a href="#">My Logo</a></div>
+      <div class="menu-nav">
+        <button class="hamburger" type="button"><img src="img/icon-hamburger.png" alt="Menu icon"></button>
+        <ul class="links">
+          <li><a>Portfolio</a></li>
+          <li><a>About</a></li>
+          <li><a>Contact</a></li>
+        </ul>
+      </div>
+    </nav>
```

Comment:
- [ ] I appreciate🎉🎉 your efforts in incorporating the <`nav>` tag into your project. To adhere to HTML5 structural guidelines, it is recommended to place the` <nav>` element inside the `<header>` tag. 
- [ ] This ensures a more semantic and organized structure for your HTML document. By following this practice, you can enhance the overall code structure and maintain compliance with HTML5 standards.